packages/esm-tb-app/src/views/patient-summary/tb-patient-summary.component.tsx

```diff
@@ -42,47 +43,52 @@ const TBSummaryOverviewList: React.FC<PatientChartProps> = ({ patientUuid }) =>
         key: 'type',
         header: t('type', 'Type'),
         encounterTypes: [config.encounterTypes.tbProgramEnrollment],
-        getObsValue: (encounter) => {
+        getObsValue: ([encounter]) => {
           return getObsFromEncounter(encounter, config.obsConcepts.type);
         },
       },
       {
         key: 'site',
         header: t('site', 'Site'),
         encounterTypes: [config.encounterTypes.tbProgramEnrollment],
-        getObsValue: (encounter) => {
+        getObsValue: ([encounter]) => {
           return getObsFromEncounter(encounter, config.obsConcepts.site);
         },
       },
       {
         key: 'drugSensitivity',
         header: t('drugSensitivity', 'Drug Sensitivity'),
         encounterTypes: [config.encounterTypes.tbProgramEnrollment],
-        getObsValue: (encounter) => {
+        getObsValue: ([encounter]) => {
           return getObsFromEncounter(encounter, config.obsConcepts.drugSensitivity);
         },
       },
       {
         key: 'regimen',
         header: t('regimen', 'Regimen'),
         encounterTypes: [config.encounterTypes.tbProgramEnrollment],
-        getObsValue: (encounter) => {
-          return getObsFromEncounter(encounter, config.obsConcepts.regimen);
+        getObsValue: ([encounter]) => {
+          const tBEnrollmentType = findObs(encounter, config.obsConcepts.tBEnrollmentType)?.value?.uuid;
+          const regimen =
+            tBEnrollmentType === config.obsConcepts.dsTBEnrollment
+              ? config.obsConcepts.dSregimen
+              : config.obsConcepts.dRregimen;
```

Comment:
similar comment about making this reusable

Reply:
Done